src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+
+import cn from 'classnames'
+
+import styles from './ProgressBar.module.css'
+import { ProgressBarProps } from './types'
+
+const clamp = (value: number, min: number, max: number): number =>
+  Math.min(Math.max(value, min), max)
+
+export const ProgressBar: React.FC<ProgressBarProps> = ({
+  className,
+  sliderClassName,
+  sliderBarClassName,
+  min = 0,
+  max = 1,
+  value,
+  showLabels = false,
+  minWrapper: MinWrapper,
+  maxWrapper: MaxWrapper
+}: ProgressBarProps) => {
+  const [sliderWidth, setSliderWidth] = useState(0)
+
+  useEffect(() => {
+    const percentage = (clamp(value - min, 0, max) * 100) / (max - min)
```

Comment:
I think the second param to clamp should be min instead of 0

Reply:
Yea this is a strange line, but it's just to get the width of the bar. With these types of props `min={3} max={5} value={3}`, this still needs to have a width of 0.